docs/concepts/configuration/organize-cluster-access-kubeconfig.md

```diff
@@ -146,7 +146,7 @@ are stored absolutely.
 {% capture whatsnext %}
 
 * [Configure Access to Multiple Clusters](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
-* [kubectl config](/docs/user-guide/kubectl/{{page.version}}/)
+* [kubectl config](/docs/reference/generated/kubectl/kubectl/)
```

Comment:
```
* [`kubectl config`](docs/reference/generated/kubectl/kubectl-commands#config)
```

Reply:
Hey @janetkuo , I'm a bit new to this process. Do you want me to add a commit with the changes in your comments, or are they comments just for future reference?

Reply:
Need a forward slash in front of "docs".
(/docs/reference/...)

Reply:
Yes, I missed a forward slash